cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-contract.help

```diff
@@ -1,43 +1,43 @@
 NAME
-       ballerina openapi  gen-contrac - generate an OpenApi contract for a given Ballerina service
+       ballerina openapi  gen-contract - generate an OpenAPI contract
+       for a given Ballerina service
 
 
 SYNOPSIS
        ballerina openapi gen-contract [<module>:]<service-name>
-       [-i <ballerina-file>]
-       [-o <contract-file>] [-s|--skip-bind]
+                                      [-i <ballerina-file>]
+                                      [-o <contract-file>] [-s|--skip-bind]
 
 
 DESCRIPTION
-       Note: This is an experimental tool ship under ballerina and will only
-       support limited set of functionality.
+       Note: This is an experimental tool which only supports a
+       limited set of functionality.
 
 
-OPTIONS
        module
-           [ Optional if ballerina file is provided ] If provided, the contract
-           will be generated using given module name.
+           [Optional] The module name to use when generating the client.
 
        service-name
-           Service name to generate the openapi contract.
+           The client name to use when generating the client.
```

Comment:
Is this correct? Shouldn't it be service name?

Reply:
Oops! yup will fix it